### PR TITLE
Support absolute import paths in less files

### PR DESCRIPTION
**Pull request by tobiasso85**
Absolute import paths are supported with this change

e.g. file
test/fixtures/libraries/scopes/comments/lib3/comments/themes/bar/library.source.less
```
@import "/lib3/comments/themes/other/sub/my.less";
```

will be passed as "/lib3/comments/themes/other/sub/my.less" to the filesystem.

Before it was interpreted as a relative path (without the leading slash ("/").

This is especially useful when the filesystem layer is abstracted such as [@ui5/fs](https://github.com/SAP/ui5-fs).

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -150,7 +150,10 @@ Builder.prototype.build = function(options) {
 	}
 
 	function fileHandler(file, currentFileInfo, handleDataAndCallCallback, callback) {
-		const pathname = path.posix.join(currentFileInfo.currentDirectory, file);
+		let pathname = path.posix.join(currentFileInfo.currentDirectory, file);
+		if (path.isAbsolute(file)) {
+			pathname = path.posix.join(file);
```
Comment (codeworrior) [inline lib/index.js]: what is this single join good for? isn't this just a normalize?

Comment (codeworrior) [reply lib/index.js]: and the join call in L153 is unnecessary for an absolute file, maybe better move it into an else branch.

Comment (codeworrior) [reply lib/index.js]: shouldn't it be path.posix.isAbsolute?

Comment (tobiasso85) [reply lib/index.js]: > what is this single join good for? isn't this just a normalize?

good point, yes [join](https://nodejs.org/api/path.html#path_path_join_paths) with a single argument is a [normalize](https://nodejs.org/api/path.html#path_path_normalize_path)

Comment (tobiasso85) [reply lib/index.js]: > shouldn't it be path.posix.isAbsolute?

done with 39df2b1

Comment (codeworrior) [reply lib/index.js]: LGTM. But I don't know the code enough for an approval.

**Pull request merged**